blockchain/metrics.go

```diff
@@ -31,6 +31,9 @@ var (
 	cacheGetBadBlockMissMeter = metrics.NewRegisteredMeter("klay/cache/get/badblock/miss", nil)
 	cacheGetBadBlockHitMeter  = metrics.NewRegisteredMeter("klay/cache/get/badblock/hit", nil)
 
+	cacheGetNodesSizeMeter     = metrics.NewRegisteredGauge("klay/cache/get/nodessize", nil)
+	cacheGetPreimagesSizeMeter = metrics.NewRegisteredGauge("klay/cache/get/preimagessize", nil)
```

Comment:
The metric type is **Gauge**, but the names are **Meter**.

Reply:
Thanks, I didn't notice it.. Changed it to Gauge.